bigframes/operations/semantics.py

```diff
@@ -49,9 +67,89 @@ def filter(self, instruction: str, model):
             ValueError: when the instruction refers to a non-existing column, or when no
                 columns are referred to.
         """
+        _validate_model(model)
+
+        output_instruction = "Based on the provided context, reply to the following claim by only True or False:"
+
+        from bigframes.dataframe import DataFrame
+
+        results = typing.cast(
+            DataFrame, model.predict(self._make_prompt(instruction, output_instruction))
+        )
+
+        return self._df[
+            results["ml_generate_text_llm_result"].str.lower().str.contains("true")
+        ]
+
+    def map(self, instruction: str, result_column_name: str, model):
```

Comment:
nit: output_column. This is trying to align with pandas style.

Reply:
Good idea!